src/axom/inlet/examples/nested_structs.cpp

```diff
@@ -327,6 +327,9 @@ int main(int argc, char** argv)
   Shape::defineSchema(shapes_container);
   // _inlet_nested_struct_array_end
 
+  // Mark the entire input as "strict"
+  inlet.getGlobalContainer().strict();
```

Comment:
Since this as an axom example, it might be better to expose this as a command line parameter.
E.g. to only treat the data as strict when the user requests it.  (Perhaps w/ a default of `true`)

This can help users see how `strict` might be made optional.